pennylane/gradients/hadamard_gradient.py

```diff
@@ -359,6 +355,18 @@ def _expval_hadamard_grad(tape, argnum, aux_wire):
 
     multi_measurements = len(tape.measurements) > 1
     multi_params = len(tape.trainable_params) > 1
+    measurements_probs = [
+        idx
+        for idx, m in enumerate(tape.measurements)
+        if isinstance(m, qml.measurements.ProbabilityMP)
+    ]
+
+    def _postprocess_probs(res, measurement, projector):
```

Comment:
I like adding this helper function! I totally would have forgotten we also need to take the wires from the measurement/tape in the `processing_fn` as well 😆.